src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly/index.js

```diff
@@ -27,11 +28,11 @@ class BaseAnchorForCommentsOnly extends React.Component {
      * Initiate file downloading and update downloading flags
      *
      * @param {String} href
-     * @param {String} fileName
+     * @param {String} displayName
      */
-    processDownload(href, fileName) {
+    processDownload(href, displayName) {
         this.setState({isDownloading: true});
-        fileDownload(href, fileName).then(() => this.setState({isDownloading: false}));
+        fileDownload(href, displayName).then(() => this.setState({isDownloading: false}));
     }
```

Comment:
No need to change here. `fileName` makes more sense here.

Reply:
Ok